src/DssSpell.t.sol

```diff
@@ -2282,35 +2456,77 @@ contract DssSpellTest is DSTest, DSMath {
+        _test_RWA_values(bytes32("RWA006-A"), addr.addr("RWA006_A_URN"), 0 * WAD, 0 * RAY, true);
+    }
+
+    function _test_RWA_values(bytes32 ilk, address urn_, uint256 price_, uint256 spot_, bool requiresLock) internal {
         jug.drip(ilk);
 
         // Confirm pip value.
-        DSValueAbstract pip = DSValueAbstract(addr.addr("PIP_RWA002"));
-        assertEq(uint256(pip.read()), 22_495_725 * WAD);
+        RwaLiquidationLike RwaLiqOracle = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
+        (,address pip_,,) = RwaLiqOracle.ilks(ilk);
+        DSValueAbstract pip = DSValueAbstract(pip_);
+        assertEq(uint256(pip.read()), price_);
 
         // Confirm Vat.ilk.spot value.
         (uint256 Art, uint256 rate, uint256 spot, uint256 line,) = vat.ilks(ilk);
-        assertEq(spot, 21_424_500 * RAY);
+        assertEq(spot, spot_);
 
         // Test that a draw can be performed.
-        address urn = addr.addr("RWA002_A_URN");
-        giveAuth(urn, address(this));
-        RwaUrnLike(urn).hope(address(this));  // become operator
+        giveAuth(urn_, address(this));
+        RwaUrnLike(urn_).hope(address(this));
+
+        if (requiresLock) {
+            address operator_ = RwaUrnLike(urn_).outputConduit();
+            giveAuth(operator_, address(this));
+            TinlakeManagerLike(operator_).file("urn", urn_);
+            TinlakeManagerLike(operator_).lock(1 ether);
+        }
+
         uint256 room = sub(line, mul(Art, rate));
         uint256 drawAmt = room / RAY;
         if (mul(divup(mul(drawAmt, RAY), rate), rate) > room) {
             drawAmt = sub(room, rate) / RAY;
         }
-        RwaUrnLike(urn).draw(drawAmt);
+
+        RwaUrnLike(urn_).draw(drawAmt);
         (Art,,,,) = vat.ilks(ilk);
         assertTrue(sub(line, mul(Art, rate)) < mul(2, rate));  // got very close to line
     }
+
+    function testManagerWards() public {
+        // RWA003-RWA005 don't have clerks deployed yet as they are existing pools
+        address RWA003_MGR = addr.addr("RWA003_A_INPUT_CONDUIT");
+        address RWA003_ROOT = 0xdB3bC9fB1893222d266762e9fF857EB74D75c7D6;
+        assertTrue(isWard(RWA003_MGR, RWA003_ROOT));
+
+        address RWA004_MGR = addr.addr("RWA004_A_INPUT_CONDUIT");
+        address RWA004_ROOT = 0x4cA805cE8EcE2E63FfC1F9f8F2731D3F48DF89Df;
+        assertTrue(isWard(RWA004_MGR, RWA004_ROOT));
+
+        address RWA005_MGR = addr.addr("RWA005_A_INPUT_CONDUIT");
+        address RWA005_ROOT = 0x4B6CA198d257D755A5275648D471FE09931b764A;
+        assertTrue(isWard(RWA005_MGR, RWA005_ROOT));
+
+        address RWA006_MGR = addr.addr("RWA006_A_INPUT_CONDUIT");
+        address RWA006_ROOT = 0xA1C16730637445D141d6ee35E31a67993FBe3759;
+        address RWA006_CLERK = 0xC5B888722069bCDFf4cAb541Da6CD09B0658d079;
+        assertTrue(isWard(RWA006_MGR, RWA006_ROOT));
+        assertTrue(isWard(RWA006_MGR, RWA006_CLERK));
```

Comment:
Why are we missing `RWA003_CLERK`, `RWA004_CLERK`, and `RWA005_CLERK`?

Reply:
Those aren't deployed yet, and will be deployed and filed in an integration spell we'll run after the executive vote. The difference between 3-5 and 6 is that the first three are existing Tinlake pools, while the last one is a new one we've deployed from scratch with the clerk already in place.